packages/wrapped-keys/src/lib/service-client/types.ts

```diff
@@ -30,6 +30,14 @@ export interface StoreKeyParams extends BaseApiParams {
   >;
 }
 
+export interface StoreKeyBatchParams extends BaseApiParams {
+  pkpAddress: string;
```

Comment:
Why is `pkpAddress` required as it isn't require in `StoreKeyParams`?

Reply:
Good catch -- see https://github.com/LIT-Protocol/js-sdk/pull/686#discussion_r1797487662 - I resolved this